components/Tabs/Tabs.js

```diff
@@ -1,52 +1,65 @@
-
 class TabLink {
-  constructor(element) {
-    // Assign this.element to the passed in DOM element
-    // this.element;
-    
-    // Get the custom data attribute on the Link
-    // this.data;
-    
-    // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
-    // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
-    // Add a click event listener on this instance, calling the select method on click
-
-  };
-
-  select() {
-    // Get all of the elements with the tabs-link class
-    // const links;
-
-    // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
-    // Add a class named "tabs-link-selected" to this link
-    // this.element;
-    
-    // Call the select method on the item associated with this link
-
-  }
+    constructor(element) {
+        // Assign this.element to the passed in DOM element
+        // this.element;
+        this.element = "element";
+
+        // Get the custom data attribute on the Link
+        // this.data;
+        this.data = this.element.dataset.tab;
```

Comment:
What you want to do here is get the custom data attribute on the Link, which you can target with the `.tabs-link` class. 

Should look something like this:
```suggestion
        this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
```

What this line does is it uses querySelector to grab the `.tabs-link` with the data-tab value equal to the data-tab value of the particular tab link clicked on. 

The value of the selected tab-links data-tab is accessed through `element.dataset.tab`.

If you were to insert a console.log below this that logged `element` (or open up the indices of any NodeList), you would see the attribute 'dataset'. When a data attribute is set, the value can be see stored in the dataset attribute. 'tab', in this context, is the name of the data attribute.